desc/continuation.py

```diff
@@ -729,17 +717,21 @@ def solve_continuation(  # noqa: C901
                 verbose=verbose,
                 copy=False,
             )
-            for arg in arg_order:
-                val = np.asarray(getattr(eqp, arg))
-                if val.size:
-                    setattr(eqi, arg, val)
+            eqi.params_dict = eqp.params_dict
             deltas = {}
             del eqp
 
         if not eqi.is_nested(msg="manual"):
             stop = True
 
         if not stop:
+            # TODO: add ability to rebind objectives
```

Comment:
what prevents this currently? does binding an objective not just say which eq to build with?